android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -78,16 +79,54 @@ public void setUserId(String userId) {
         Radar.setUserId(userId);
     }
 
+    @ReactMethod
+    public void getUserId(final Promise promise) {
```

Comment:
Can you add test for all of the new functions here? https://github.com/radarlabs/react-native-radar/blob/master/test/index.test.js